examples/smartcore-ml/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 risc0-zkvm = { path = "../../risc0/zkvm" }
 rmp-serde = "1.1"
 serde_json = "1.0"
-smartcore = { version = "0.3", features = ["serde"] }
+smartcore = { git = "https://github.com/risc0/smartcore.git", features = ["serde"] }
```

Comment:
we should use a git reference

Reply:
I've not pinned to a particular git commit and added a note about why the git reference is used instead of using the published crate